pkg/api/bzz_test.go

```diff
@@ -32,144 +30,392 @@ import (
 	"github.com/ethersphere/bee/pkg/tags"
 )
 
-func TestBzz(t *testing.T) {
+func TestBzzFiles(t *testing.T) {
 	var (
-		bzzDownloadResource = func(addr, path string) string { return "/bzz/" + addr + "/" + path }
-		storer              = smock.NewStorer()
-		ctx                 = context.Background()
-		mockStatestore      = statestore.NewStateStore()
-		logger              = logging.New(ioutil.Discard, 0)
-		client, _, _        = newTestServer(t, testServerOptions{
-			Storer: storer,
+		fileUploadResource   = "/bzz"
+		targets              = "0x222"
```

Comment:
invalid hex?

Reply:
I actually did not notice this. The test doesn't verify this. From the API perspective, all we need to check is that the value we are returned in the header is the same as the value we sent. This is why the test passes. Not really sure where the targets are being used. Again this test was moved from the files_test.go file. I have retained most of it. Previously they were using this value.

Reply:
The API won't verify the hex value here, as it is passed to netstore

Reply:
This is not required for the test logic. So keeping it as is for now.